test/e2e/config/packet-ci-actions.yaml

```diff
@@ -114,7 +114,7 @@ providers:
       # Add cluster templates
       - sourcePath: "../data/v1alpha3/cluster-template-packet-ccm.yaml"
       - sourcePath: "../data/v1alpha3/cluster-template-cpem.yaml"
-  - name: v0.5.0
+  - name: v0.6.0
```

Comment:
It might be a good idea to leave an entry in here for the v0.5.0 release pointing at the release URL, similar to the v0.3.11 release above. That would allow you to add an e2e test that validates an upgrade from v0.5.0 to v0.6.0.